### DISCUSSION TITLE
Spatial and temporal reference systems are directive

### DISCUSSION THREAD

**Discussion by alanruttenberg** [CommonCoreOntology/CommonCoreOntologies#892] [Ontology Terms and Class Schema]
Currently, spatial reference system and temporal reference systems are considered descriptive. I'd argue that they are, as *systems*, directive. A system for assigning times or spatial coordinates is a recipe for how to create a coordinate. It *prescribes* the coordinate, which in itself is sometimes descriptive and sometimes designative. An example of designative use is would be to identify a feature by it's rough coordinate. An example of descriptive use would be "The bounds of this lot are roughly \<bounding box\>". 

The changes I advocate for are:

- Move temporal and spatial reference system terms to be children of directive information entity.
- Either deprecate 'is reference system of' and 'is geospatial coordinate reference system of' (preferred, since fewer relations are better) or make them subproperties of 'is prescribed by'. Similar their inverses.

There are other requests that I will likely make related to coordinate systems. An example would be to move most if not all geospatial spatial regions to be boundaries, as spatial regions can bear no qualities (but need to be able to), and because there is a discord between, for example, a geospatial ellipse being a boundary, but it's major and minor axes not.

**Comment by cameronmore** [comment]
On the one hand, systems like coordinates and zip codes are prescriptive. On the other hand, the system was imposed with the intention to describe the area. In order to describe anything, language must be prescribed, or imposed, on the things being referred to. This is a tricky philosophical issue. Does the word 'apple' merely describe something, or is it a prescription that humans are projecting out onto the world? I think we would want to say the later, and I think this is the same intuition we have for zip codes and coordinates.

However, certain coordinates that direct people where to do may prescribe locations and processes occurring at those locations, but I think the actual coordinates in a directive ICE are descriptive parts of the larger ICE. If an ICE directs someone to do something at some coordinates, the coordinates are not the prescriptive part, they are describing a place where one is instructed to go.

I'm not aiming to refute, just wanted to give this issue some attention and encourage discussion, open to other ideas, like Tim's.

**Comment by alanruttenberg** [comment]
A coordinate reference *system* is like a recipe. It's a method to assign a name to a thing. 

Your example of "apple", goes in the other direction, questioning whether descriptive things should be considered prescriptive. 

I might ultimately want you to be able to describe what an apple cake is, but if I go about that by giving you a recipe to make one, the ICE is directive. The ICE isn't describing an apple cake looks like, it's describing how to make one.

Or suppose I want to be able to describe a plot of land, so I hire a surveyor. The surveyor follows a procedure represented as a directive in order to compute the parameters of the lot. Part of the procedure is proper use of the coordinate reference system, for instance an understanding of where reference points are and how to calculate a coordinate based on the known coordinates of reference points.
